src/views/index.ejs

```diff
@@ -130,10 +132,16 @@
         document.getSelection().removeAllRanges();
         document.getSelection().addRange(selected);
       }
-      setInterval(function() {
-        this.textContent = "COPIED!"
-      }, 3000);
+
+      
     };
+    /*Tippy Tooltip*/
+tippy('.tippy, [data-tippy-content], [data-tooltip]', {
+    // flipOnUpdate: true,
+    // boundary: 'window',
+    trigger: 'click'
+});
```

Comment:
This block should come after the whole copy function rather than coming midway before the reach part that does the copy.